Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml.cs

```diff
@@ -116,7 +116,7 @@ async void RunMBaaSAsync(object sender, EventArgs e)
             }
             try
             {
-                var list = await Data.ListAsync<CustomDocument>(DefaultPartitions.UserDocuments);
+                var list = await Data.ListAsync<CustomDocument>(DefaultPartitions.UserDocuments, new ReadOptions());
```

Comment:
Does not seem related to the changes in this PR, we should probably not commit that.

Reply:
I think it's an OK fix, since the API supports it.  No harm done in this case since there is no functionality change

Reply:
But there is no fix here.